components/src/Modal.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+import { Modal } from "hanheel-modal-payments";
+import React from "react";
+
+const meta: Meta<typeof Modal> = {
+  title: "Components/Modal",
+  component: Modal,
+  argTypes: {
+    position: {
+      control: { type: "radio" },
+      options: ["center", "bottom"],
+    },
+  },
+};
+export default meta;
+
+type Story = StoryObj<typeof Modal>;
+
+export const Default: Story = {
+  render: (args) => {
+    const [isOpen, setIsOpen] = useState(true);
+
+    const handleCloseModal = () => {
+      setIsOpen(false);
+    };
+
+    return (
+      <>
+        <button onClick={() => setIsOpen(true)}>Open Modal</button>
+        <Modal {...args} isOpen={isOpen} handleCloseModal={handleCloseModal}>
+          <div style={{ padding: "20px" }}>모달 content 입니다</div>
+        </Modal>
+      </>
+    );
+  },
+  args: {
+    title: "모달 타이틀",
+    position: "center",
+  },
+};
```

Comment:
컴포넌트의 사용 예시를 스토리에 밝힌 것은 좋습니다. 그러나 Modal 컴포넌트의 isOpen과 handleCloseModal props 명세가 드러나지 않아 해당 컴포넌트를 사용자가 비제어 컴포넌트로 오해할 여지가 있습니다. 또한 현재의 명세는 다소 단순하므로 아래와 같이 구성하여 이용자가 컴포넌트의 활용 방법을 더 쉽게 이해할 수 있도록 구성하면 좋을 것 같습니다.

```suggestion
import type { Meta, StoryObj } from "@storybook/react";
import Modal from "./lib/Modal";

const meta: Meta<typeof Modal> = {
  title: "Components/Modal",
  component: Modal,
};

export default meta;
type Story = StoryObj<typeof Modal>;

// 기본 모달
export const Default: Story = {
  args: {
    isOpen: true,
    children: <div style={{ padding: "20px" }}>모달 content 입니다</div>,
  },
};

// 하단 모달
export const BottomModal: Story = {
  args: {
    isOpen: true,
    position: "bottom",
    children: <div style={{ padding: "20px" }}>모달 content 입니다</div>,
  },
};

```

Reply:
Storybook을 문서화 도구로서 사용하는 방법이군요..! props 명세와 사용 시나리오를 명확히 보여줄 수 있도록 개선했습니다! 감사합니다 :)